scripts/_lines_of_code.py

```diff
@@ -45,50 +45,90 @@ def execute_bind(id_, message):
 class NotContributedError(Exception):
     pass
 
-
+excepted_files = [
+    'package.json',
+    'package-lock.json',
+    '.gitignore',
+    'yarn.lock'
+]
 class ContributedRepository:
     def __init__(self, contributed_to: Repository, user):
         self.total_lines_of_addition_in_contribution = 0
         self.total_lines_of_deletion_in_contribution = 0
+        self.excepted_lines_of_addition_in_contribution = 0
+        self.excepted_lines_of_deletion_in_contribution = 0
         self.name = contributed_to.full_name
+
         for contribution in contributed_to.get_stats_contributors():
             if contribution.author.id == user.id:
                 for week in contribution.weeks:
                     self.total_lines_of_addition_in_contribution += week.a
-                    self.total_lines_of_deletion_in_contribution += week.d
+                    self.total_lines_of_deletion_in_contribution += week.dRepository
         if not any([self.total_lines_of_addition_in_contribution, self.total_lines_of_deletion_in_contribution]):
             raise NotContributedError('User have not contributed to this repository.')
+        
+        for commit in contributed_to.get_commits():
+            if commit.author.id == user.id:
+                for file in commit.files:
+                    if file.filename in excepted_files:
+                        self.excepted_lines_of_addition_in_contribution += file.additions
+                        self.excepted_lines_of_deletion_in_contribution += file.deletions
+
+        self.net_addition_in_contribution = self.total_lines_of_addition_in_contribution - self.excepted_lines_of_addition_in_contribution
+        self.net_deletion_in_contribution = self.total_lines_of_deletion_in_contribution - self.excepted_lines_of_deletion_in_contribution
 
 
 class OwnedRepository:
     def __init__(self, owned_repository: Repository, user):
         self.total_lines_of_addition = 0
         self.total_lines_of_deletion = 0
+        self.excepted_lines_of_addition = 0
+        self.excepted_lines_of_deletion = 0
         self.name = owned_repository.name
+
         for contribution in owned_repository.get_stats_contributors():
             if contribution.author.id == user.id:
                 for week in contribution.weeks:
                     self.total_lines_of_addition += week.a
                     self.total_lines_of_deletion += week.d
 
+        if (owned_repository.get_commits().totalCount >= 100):
+            print('This repository has a lot of commits! This can take some time...')
```

Comment:
Use ```execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')``` instead of ```print('This repository has a lot of commits! This can take some time...')```. (I am planning to create GUI and web versions after adding some features...) print statements should not be used then...